rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -752,6 +752,26 @@ namespace rsx
 		return result;
 	}
 
+	static inline f32 decode_fx13(u32 bits)
+	{
+		// Classic fixed point, see PGRAPH section of nouveau docs for TEX_FILTER (lod_bias) and TEX_CONTROL (min_lod, max_lod)
+		// Technically min/max lod are fixed 4.8 but a 5.8 decoder should work just as well since sign bit is 0
+
+		f32 base;
+		if ((bits & (1 << 12)) == 0)
+		{
+			base = f32(bits >> 8);
+		}
+		else
+		{
+			bits = ~bits + 1;
+			base = -f32(bits >> 8);
+		}
+
+		const auto frc = (bits & 0xff) / 255.f;
```

Comment:
This is a little odd, so `0x1.ff` (as hexfloat) is equal to 2.0 here?
If not this needs to be divided by 256.

Reply:
Good catch, I forgot to carry.